File: 0485-max-consecutive-ones/0485-max-consecutive-ones.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        n=len(nums)
        temp=0
        count=0
        for i in range(0,n):
            if(nums[i]==1):
                count=count+1
                temp=max(count,temp)
            else:
                count=0
        
        return temp
        

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int n=nums.size();
        int count=0;
         int temp=-1;
        for(int i=0;i<n;i++){
        
            if(nums[i]==1){
                count++;
            temp=max(temp,count);
            }
            else{
            count =0;
            }
        }
        return temp;
    }
};